app/uk/gov/hmrc/gform/fileupload/FileUploadConnector.scala

```diff
@@ -27,9 +27,10 @@ import uk.gov.hmrc.gform.sharedmodel.formtemplate.FormTemplateId
 import uk.gov.hmrc.gform.time.TimeProvider
 import uk.gov.hmrc.gform.wshttp.{ FutureHttpResponseSyntax, WSHttp }
 import uk.gov.hmrc.http.{ HeaderCarrier, HttpResponse }
-import uk.gov.hmrc.play.http.logging.MdcLoggingExecutionContext._
 
 class FileUploadConnector(config: FUConfig, wSHttp: WSHttp, timeProvider: TimeProvider) {
+  implicit val ec = play.api.libs.concurrent.Execution.defaultContext
```

Comment:
May we please get rid of all 
```
implicit val ec = play.api.libs.concurrent.Execution.defaultContext
```
and pass execution context as `implicit` parameter as I did on gform-frontend.

In that way we have only one place where we choose which `ExecutionContext` to use.